samples/meeting-live-coding-interview/csharp/MeetingLiveCoding/ClientApp/src/types/question.ts

```diff
@@ -0,0 +1,25 @@
+export const IQuestionDetails = {
+    questions: [
+        {
+            srNo: 1,
```

Comment:
What is this srNo meant to mean? when I see it across the code I get confused by what it represents. Could we just change it to questionId?